android/shared_config.gradle

```diff
@@ -0,0 +1,27 @@
+import groovy.json.JsonSlurper
+
+/*
+ * This function returns the relevant React Native version. Before doing so,
+ * it performs a check to insure that the javascript version for React Native
```

Comment:
> javascript version for React Native

I assume that by "javascript" version you mean the version defined in the package.json, right? How about naming that "the package.json version of RN"? I don't think mentioning "JavaScript" is relevant here anyway, the package.json is a npm/Node construct anyway.

If we rename here, we should also rename the variable names in the code below too.

Reply:
👍 Makes sense to me. Updated.